modules/ROOT/pages/database-administration/index.adoc

```diff
@@ -53,8 +53,8 @@ It is different from the default database, which is the database that the server
 Per-user home databases are controlled via the Cypher administration commands.
 
 To set a home database for a user, this user must exist as a record in Neo4j.
-Therefore, for deployments using xref:authentication-authorization/introduction.adoc[auth providers] other than native, you create a native user with a matching username and then set a home database for that user.
-For more information on creating native users and configuring a home database for a user, see link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/manage-users[Cypher Manual -> User Management].
+Therefore, for deployments using xref:authentication-authorization/index.adoc[auth providers] other than native, you create a native user with a matching username and then set a home database for that user.
+For more information on creating native users and configuring a home database for a user, see xref:authentication-authorization/manage-users.adoc[Cypher Manual -> User Management].
```

Comment:
if the link no longer points to Cypher Manual, should the link text also be updated?
```suggestion
For more information on creating native users and configuring a home database for a user, see xref:authentication-authorization/manage-users.adoc[User Management].
```